pkg/webui/console/views/device/index.js

```diff
@@ -147,6 +158,12 @@ export default class Device extends React.Component {
     env: undefined,
   }
 
+  componentDidMount() {
+    const { appId, getLink } = this.props
+
+    getLink(appId, ['skip_payload_crypto'])
+  }
+
```

Comment:
Not necessary to do that here. You can add the dispatch to the `withRequest` HOC on LL94:

```
getDevice(appId, devId, selector, { ignoreNotFound: true })
getLink(appId, ['skip_payload_crypto'])
```

Or you can make a `loadDeviceData` prop in the redux connector that combines the actions. We do that e.g. in `/console/views/application/index.js LL79`.